plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -550,7 +550,8 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
                 jdbcClient.getColumns(session, handle).stream()
                         .map(JdbcColumnHandle::getColumnMetadata)
                         .collect(toImmutableList()),
-                jdbcClient.getTableProperties(session, handle));
+                jdbcClient.getTableProperties(session, handle),
+                handle.getComment());
```

Comment:
Table comment doesn't nee to be part of table handle.

Reply:
@findepi What's your suggestion? Adding an extra method to `jdbcClient`?

Reply:
Seems like this would work, right?

cc @hashhar

Reply:
It does add a new remote call but feels cleaner to me anyway.

Reply:
The separate remote call is there anyway.

Reply:
Didn't have to be. For example we already look at list of columns from the handle if present instead of calling `getColumns`.